### DISCUSSION TITLE
Fixed IP

### DISCUSSION THREAD

**Discussion by bensuffolk** [HASwitchPlate/openHASP#336] [Q&A]
Is there a way to configure fixed network Params for an instance where there is not a dhcp server on the network?

**Comment by fvanroie** [comment]
I have experimented with [setting a fixed IP in the firmware](https://github.com/HASwitchPlate/openHASP/blob/1c86a3a34fdae9af21e09272f47817895ff89250/src/sys/net/hasp_wifi.cpp#L445-L451) and it seemed to work.
This hasn't been tested thoroughly and isn't exposed yet in the Web UI, but you can give it a try.
Then we can add it in the settings later.

**Reply by nagyrobi** [reply]
Would be nice, at least as user_override

**Reply by bensuffolk** [reply]
Yes hard coding it works fine. 

Without a DHCP server running this no longer times out looking for the server and connects instantly.
